src/main/java/com/microsoft/azure/spark/tools/job/LivySparkBatchFactory.java

```diff
@@ -62,6 +62,9 @@ public LivySparkBatchFactory controlSubject(final Observer<Pair<MessageInfoType,
         logAsCtrlSubject.subscribe(
                 typedMessage -> {
                     switch (typedMessage.getFirst()) {
+                        case Debug:
+                            log.debug(typedMessage.getRight());
```

Comment:
Seems like the implemented interface `Logger` is not used?

Reply:
Fix it.